### PR TITLE
Add little padding for puzzle storm end statistics

### PR DESCRIPTION
**Pull request by MarcinK50**
My pull request adds little padding for text in puzzle storm end stats: 
photo

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -5,7 +5,6 @@
   "requires": true,
   "packages": {
     "": {
-      "name": "prettierlichess",
```
Comment (kieferro) [inline package-lock.json:8]: Why did you delete the name here?

File: src/styles.css [hunk 2]
```diff
@@ -1968,6 +1968,10 @@ button.rematch-decline::before {
     font-size: 1.5em !important;
 }
 
+.storm--end__stats th {
+    padding: 10px;
+}
+
```
Comment (kieferro) [inline src/styles.css:1974]:
```suggestion
.storm--end__stats th {
    padding-left: 10px !important;
}

```
I would prefer `padding-left` here because it is really only about the padding to the left and not the global padding. Also, as always, we should use `!important` to make sure that the rule is really applied.

**Review by kieferro — COMMENTED** [review]
Thank you very much for your contribution. It definitely makes sense to increase the padding at this point. However, I still have two small comments.

File: package-lock.json (hunk 1 above)
Comment (MarcinK50) [reply package-lock.json:8]: I really don't know why this line deleted

**Review by kieferro — APPROVED** [review]
Thank you for your contributions!
